judo-tatami-jsl-jsl2ui/src/main/epsilon/transformations/ui/modules/view/menuLinkDeclarationViewPage.etl

```diff
@@ -24,6 +24,8 @@ rule AccessViewPageDefinition
         log.info("===================");
         */
 
+        var allActionDeclarations = s.referenceType.getAllActionDeclarations();
```

Comment:
I suggest renaming this (and other occurrences) to `actionDeclarationsToProcess` or something similar to show that this collection of items is not connected to an actual instance and will be modified as items are processed during transformation.

Reply:
Why would you make a variable if you do not plan to process it?

Reply:
How do you know whether it's a variable or a const (if there's any, i don't know :D) unless you scroll back to the definition? I don't think there are const types instead of vars, in which case a calculated value could be stored in a var to prevent calculating the same multiple times